src/chainparams.cpp

```diff
@@ -78,16 +78,18 @@ class CMainParams : public CChainParams {
         consensus.nPowTargetSpacing = 10 * 60;
         consensus.fPowAllowMinDifficultyBlocks = false;
         consensus.fPowNoRetargeting = false;
-        consensus.nRuleChangeActivationThreshold = 1916; // 95% of 2016
+        consensus.nRuleChangeActivationThreshold = 1815; // 90% of 2016
```

Comment:
ACK using 1815 as the constant.

1814 has lower error from 90% than 1815, but previously we used 1916 which has greater error from 95% than 1915 but is the smallest exceeds threshold number, so we continue the same precedent (not that it matters much).